Emby.Drawing/ImageProcessor.cs

```diff
@@ -395,7 +398,7 @@ private string GetMimeType(ImageFormat format, string path)
         /// <summary>
         /// Gets the cache file path based on a set of parameters
         /// </summary>
-        private string GetCacheFilePath(string originalPath, ImageSize outputSize, int quality, DateTime dateModified, ImageFormat format, bool addPlayedIndicator, double percentPlayed, int? unwatchedCount, int? blur, string backgroundColor, string foregroundLayer)
+        private string GetCacheFilePath(string originalPath, MediaBrowser.Model.Drawing.ImageSize outputSize, int quality, DateTime dateModified, ImageFormat format, bool addPlayedIndicator, double percentPlayed, int? unwatchedCount, int? blur, string backgroundColor, string foregroundLayer)
```

Comment:
It would also be nice to split the arguments on newlines so they are easier to read here :)